BookingsApi/BookingsApi.Contract/Responses/HearingAllocationsResponse.cs

```diff
@@ -46,4 +46,9 @@ public class HearingAllocationsResponse
     /// True if the allocated user has a working hours clash
     /// </summary>
     public bool? HasWorkHoursClash { get; set; }
+    
+    /// <summary>
+    /// True if the allocated user has more than 3 concurrent hearings allocated
```

Comment:
Update the description to what the property reflect now since this is a nullable int instead of a bool

Reply:
Done